docs/index.md

```diff
@@ -53,13 +54,14 @@ class Settings(BaseSettings):
     # export my_prefix_more_settings='{"foo": "x", "apple": 1}'
     more_settings: SubModel = SubModel()
 
-    model_config = ConfigDict(env_prefix = 'my_prefix_')  # defaults to no prefix, i.e. ""
+    model_config = ConfigDict(env_prefix='my_prefix_')  # defaults to no prefix, i.e. ""
 
-print(Settings().model_dump())
+
+pprint(Settings().model_dump())
```

Comment:
i think you should just use `print`, pytest-examples should take care of formatting the output with `black`.

Reply:
Didn't work with `print`.

```
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― test_docs_examples[docs/index.md:14-72] ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
Print output changed code:
  --- before
  +++ after
  @@ -56,30 +56,21 @@
       model_config = ConfigDict(env_prefix='my_prefix_')  # defaults to no prefix, i.e. ""
 
 
   print(Settings().model_dump())
   """
  -{
  -    'auth_key': '',
  -    'api_key': '',
  -    'redis_dsn': Url('redis://user:pass@localhost:6379/1'),
  -    'pg_dsn': Url('postgres://user:pass@localhost:5432/foobar'),
  -    'amqp_dsn': Url('amqp://user:pass@localhost:5672/'),
  -    'special_function': <built-in function cos>,
  -    'domains': set(),
  -    'more_settings': {'foo': 'bar', 'apple': 1},
  -}
  +{'auth_key': '', 'api_key': '', 'redis_dsn': Url('redis://user:pass@localhost:6379/1'), 'pg_dsn': Url('postgres://user:pass@localhost:5432/foobar'), 'amqp_dsn': Url('amqp://user:pass@localhost:5672/'), 'special_function': <built-in function cos>, 'domains': set(), 'more_settings': {'foo': 'bar', 'apple': 1}}
   """
```

Reply:
Ok, I've replaced it with `print` but it doesn't work with formatted output.
we can ignore it for now.